src/Compute/Compute/Generated/Snapshot/SnapshotRevokeAccessMethod.cs

```diff
@@ -45,7 +45,15 @@ public override void ExecuteCmdlet()
                     string resourceGroupName = this.ResourceGroupName;
                     string snapshotName = this.SnapshotName;
 
-                    var result = SnapshotsClient.RevokeAccessWithHttpMessagesAsync(resourceGroupName, snapshotName).GetAwaiter().GetResult();
+                    Rest.Azure.AzureOperationResponse result;
+                    if (NoWait.IsPresent)
+                    {
+                        result = SnapshotsClient.BeginRevokeAccessWithHttpMessagesAsync(resourceGroupName, snapshotName).GetAwaiter().GetResult();
+                    }
+                    else
+                    {
+                        result = SnapshotsClient.RevokeAccessWithHttpMessagesAsync(resourceGroupName, snapshotName).GetAwaiter().GetResult();
+                    }
```

Comment:
I don't think the code below setting Uput.Name will work with the request used in the Bgein call.  Remove this for now and we cna discuss with Compute

Reply:
Ok. Done.